_includes/scripts.html

```diff
@@ -1,2 +1,3 @@
 <script src="{{ site.baseurl }}/assets/js/vendor/uswds.min.js"></script>
-<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
\ No newline at end of file
+<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
+<script async id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOC&amp;subagency=CEN" type="text/javascript"></script>
```

Comment:
I think this should be moved to `head.html` so that it's loaded within the `head` HTML tags.

Reply:
@curt-mitch-census I think I fixed this issue and would appreciate input!

Reply:
@sadiejay Sorry, I should have been more specific: you were right to move the dap script to the `head.html`, according to their [suggested documentation](https://github.com/digital-analytics-program/gov-wide-code/blob/master/documentation/GSA%20DAP%204.1%20-%20Quick%20Guide.pdf). However, I think you can keep the `styleguide.js` file within the `scripts.html` file along with the `uswds.min.js` file to keep the non-analytics JavaScript files in one place. My apologies for the mixup!